src/main/java/Duke/Deadline.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+
+public class Deadline extends Task {
+    private final LocalDateTime dueDate;
+
+    Deadline(String name, String dueDate) throws DateTimeParseException {
```

Comment:
Perhaps including access modifiers here would have made the code easier to read.

Reply:
the default access modifier refers to a package private constructor here, which is intended!